### ISSUE TITLE
Cannot override hardcoded arguments

### ISSUE THREAD

**Issue by disconn3ct** [gabe565/charts#544]
### Chart Name

headscale

### Chart Version

0.13.0

### Bug Description

The container image is changeable, but the args are not. This means that (for example) `ghcr.io/juanfont/headscale` does not work as of 0.23.0 (alpha)

### values.yaml

```yaml
image:
  repository: "ghcr.io/juanfont/headscale"
  tag: "0.23.0" # 0.22.3 works
```

### Relevant log output

```shell
Error: unknown command "headscale" for "headscale"
Run 'headscale --help' for usage.
```

**Comment by gabe565** [comment]
@disconn3ct Hmm you're right. Let me move that from `hardcodedValues` to `values.yaml`.

**Comment by gabe565** [comment]
@disconn3ct I just released v0.13.1 of the Headscale chart which should fix this! Can you test again with
```yaml
args: [serve]
```

**Comment by disconn3ct** [comment]
Looks good!
